### PR TITLE
PR: Add a command line option to load projects 

### LINKED ISSUE #3830: Add a command line option to load projects
Is there an option to load a specific project with command line options? Or, alternatively, specify a  different set of configuration files? The reason is that we are working with multiple users on the same account, and we do not want to have our projects and settings mixed.

We already have different spyder versions installed using environments (with conda, windows), but these different spyder versions all use the same configuration files.

## Versions and main components

* Spyder Version: 3.0.2
* Python Version: 3.5
* Operating system: windows

### PR DESCRIPTION
**Pull request by andfoy**
Fixes #3830 
The flag option to use this function is ```--project <path>```

### CODE REVIEW

File: spyder/app/cli_options.py [hunk 1]
```diff
@@ -37,5 +37,8 @@ def get_options():
                            "not related with Python profiling)")
     parser.add_option('--window-title', type=str, default=None,
                       help="String to show in the main window title")
+    parser.add_option('--project', default=None, type=str,
```
Comment (goanpeca) [inline spyder/app/cli_options.py]: Could we add a short flag as well?

`-p` what do you think @ccordoba12?

Comment (ccordoba12) [inline spyder/app/cli_options.py]: Yeah, why not?

Comment (goanpeca) [inline spyder/app/cli_options.py]: Do we have a way of opening a file? that would be useful as well

spyder somefile.py

Comment (ccordoba12) [inline spyder/app/cli_options.py]: Yes, it has been there since Spyder 2.2 :-)

Comment (goanpeca) [inline spyder/app/cli_options.py]: Ok good :-)

**Comment by ccordoba12** [comment]
This looks to me, thanks @andfoy!

**Pull request merged**